MusicNotationKit/MusicNotationKit/Clef.swift

```diff
@@ -13,46 +13,48 @@ public struct Clef {
      Starts from 0 on the first line (from the bottom). Ledger lines below that are negative.
      Each increase by 1 moves a half step. i.e. 1 is the first space on the staff.
      */
-    public let lineNumber: Int
+    internal var halfSteps: Int {
```

Comment:
Since `staffLocation` is immutable, this could be a `let` and set at initialization.